lib/internal/test_runner/runner.js

```diff
@@ -345,7 +345,6 @@ function runTestFile(path, filesWatcher, opts) {
 
     let err;
 
-
```

Comment:
This extra line has been bothering me for so long 😄

Reply:
same